src/convertToObject.js

```diff
@@ -3,16 +3,44 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const copyOfSourceString = sourceString.replace(/\n/g, '').trim().split(' ');
+  const copyOfSourceStringWithoutSpaces = copyOfSourceString
+    .filter(item => item !== '')
+    .join(' ').replace(/(\s+)(:)/g, '$2').replace(/\s*;\s*/g, ';')
```

Comment:
![image](https://github.com/mate-academy/js_style-to-object/assets/96409563/956146c1-070a-43c5-9048-740ea2563fb6)